python/src/gen3authz/client/arborist/base.py

```diff
@@ -622,6 +623,23 @@ async def update_policy(self, policy_id, policy_json, create_if_not_exist=False)
         self.logger.info("put policy {}".format(policy_id))
         return response
 
+    @maybe_sync
+    async def update_bulk_policy(self, policy_json, create_if_not_exist=False):
+        try:
+            url = self._bulk_policy_url
+            response = await self.put(url, json=policy_json)
+        except ArboristError as e:
+            if e.code == 405:
+                # For compatibility with Arborist 2.x.x
+                self.logger.info(
+                    "This Arborist version has no PUT /policy/{policyID} endpt yet."
+                    "Falling back on PUT /policy"
+                )
+                response = await self.put(self._policy_url, json=policy_json)
+            else:
+                raise
```

Comment:
i don't think the try/except is necessary.
if you do want backwards compatibility, you'll have to make multiple calls for each policy in `policy_json`, right?

Reply:
Yeah it doesnt evenn matter if its backwards compatible. This is a completely different function. I just copy pasted and forgot to clean up :grimacing: